batchflow/research/job.py

```diff
@@ -6,19 +6,19 @@
 
 class Job:
     """ Contains one job. """
-    def __init__(self, executable_units, n_iters, repetition, configs, branches, name):
+    def __init__(self, executable_units, n_iters, repetition, configs, cv_splits, branches, name):
```

Comment:
```suggestion
    def __init__(self, executable_units, n_iters, repetition, configs, cv, branches, name):
```

Reply:
It is not clear: `cv` can be the number of folds or indexes of folds so in the first case I use `n_splits` and in the second `cv_splits`. Maybe we should use `folds` for that parameter.